draft-ietf-quic-tls.md

```diff
@@ -1536,14 +1536,7 @@ After this period, old read keys and their corresponding secrets SHOULD be
 discarded.
 
 
-## Minimum Key Update Frequency
-
-Key updates MUST be initiated before usage limits on packet protection keys are
-exceeded. For the cipher suites mentioned in this document, the limits in
-Section 5.5 of {{!TLS13}} apply. {{!TLS13}} does not specify a limit for
-AEAD_AES_128_CCM, but the analysis in {{ccm-bounds}} shows that a limit of 2^23
-packets can be used to obtain the same confidentiality protection as the limits
-specified in TLS.
+## Limits on Key Usage
 
 The usage limits defined in TLS 1.3 exist for protection against attacks
```

Comment:
This still jumps in a bit cold.

```suggestion
This document sets usage limits for AEAD keys to ensure that overuse of a single key does
not give an adversary a disproportionate advantage in attacking the confidentiality
and integrity of communications.

The usage limits defined in TLS 1.3 exist for protection against attacks
```

Or some such nonsense.

Reply:
I'm happy with that as an intro, though might tweak it a bit to emphasize that limits don't always apply to keys (but to connections).